core/templates/pages/collection-player-page/collection-player-page.directive.html

```diff
@@ -1,6 +1,11 @@
 <div>
   <background-banner></background-banner>
-  <div ng-if="$ctrl.collection" class="oppia-collection-player-tiles-container">
+  <div ng-if="!$ctrl.explorationAdded" class="no-exploration-created">
```

Comment:
Hm would !$ctrl.collection.getCollectionNodeCount() be sufficient?

Reply:
Done.